combat-trainer.lic

```diff
@@ -2065,6 +2065,17 @@ class TrainerProcess
           fix_standing
         end
       end
+    when 'Ret Stealth'
+      retreat
```

Comment:
I had to backtrain stealth on my Barb and I was toying with something similar by extending Stealth to take an optional yaml parameter for "hide_range" with a default of melee. And if you set pole or missile, then it would ensure you were retreated to the appropriate range before hiding.